src/components/Header/Header.js

```diff
@@ -45,10 +45,8 @@ const Header = props => (
         <div className={style.subtitle}>Framework Dashboard</div>
       </div>
       <div className={style.orgContainer}>
-        <a className={style.orgLink} href={LINKS.ORG}>
```

Comment:
Also remove the associated styles, constants and assets, it they are no longer used?